### PR TITLE
AutoML: Add `plot_confusion_matrix` and `display_model_evaluation` to TablesClient.

### PR DESCRIPTION
**Pull request by jonathan1920**
…s to setup.py and noxfile.py

### CODE REVIEW

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py [hunk 1]
```diff
@@ -2778,3 +2807,256 @@ def batch_predict(
...
+                            round(confidence_metrics_entry.f1_score * 100, 2)
+                        )
+                    )
+            cm = []
+            for row in classification_metrics.confusion_matrix.row:
+                cm.append(row.example_count)
+            classes = classification_metrics.confusion_matrix.display_name
+            self.plot_confusion_matrix(np.asarray(cm), classes, True)
+
+            # print(classification_metrics.confusion_matrix.display_name)
```
Comment (busunkim96) [inline automl/google/cloud/automl_v1beta1/tables/tables_client.py]: Please un-comment or remove these lines.

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py [hunk 2]
```diff
@@ -2778,3 +2807,256 @@ def batch_predict(
...
+            >>>
+            >>> client = automl_v1beta1.TablesClient(
+            ...     credentials=service_account.Credentials.from_service_account_file('~/.gcp/account.json')
+            ...     project='my-project', region='us-central1')
+            ...
+            >>> d = client.plot_confusion_matrix(cm, ['blue','red','green'])
+            >>>
+
+        Args:
+            confusion_matrix ([[float]]):
```
Comment (busunkim96) [inline automl/google/cloud/automl_v1beta1/tables/tables_client.py]: Hmm, I'm not sure how type hints works for numpy arrays. BigQuery might have some examples?

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py [hunk 3]
```diff
@@ -2778,3 +2807,256 @@ def batch_predict(
...
+                This is a numpy array of the confusion matrix.
+            classes ([string]):
+                A list of the possible classification values.
+            title (Optional[string]):
+                The title of the confusion matrix.
+            cmap (optional[Color]):
```
Comment (busunkim96) [inline automl/google/cloud/automl_v1beta1/tables/tables_client.py]: If there is a `Color` type, where is it imported from?

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py (hunk 1 above)
Comment (jonathan1920) [reply automl/google/cloud/automl_v1beta1/tables/tables_client.py]: Will do!

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py (hunk 3 above)
Comment (jonathan1920) [reply automl/google/cloud/automl_v1beta1/tables/tables_client.py]: It is imported as a part of matplotlib

Comment (busunkim96) [reply automl/google/cloud/automl_v1beta1/tables/tables_client.py]:
```suggestion
            cmap (Optional[matpotlib.colors.Colormap]):
```

File: automl/google/cloud/automl_v1beta1/tables/tables_client.py [hunk 4]
```diff
@@ -2778,3 +2807,253 @@ def batch_predict(
...
+        Example:
+            >>> from google.cloud import automl_v1beta1
+            >>>
+            >>> from google.oauth2 import service_account
+            >>>
+            >>> client = automl_v1beta1.TablesClient(
+            ...     credentials=service_account.Credentials.from_service_account_file('~/.gcp/account.json')
+            ...     project='my-project', region='us-central1')
+            ...
+            >>> d = client.plot_confusion_matrix(cm, ['blue','red','green'])
```
Comment (sirtorry) [inline automl/google/cloud/automl_v1beta1/tables/tables_client.py]: in this case, blue/red/green are confusing class names considering colormap is an option

**Comment by busunkim96** [comment]
@jonathan1920 @helinwang

Friendly ping. Is this still being worked on?

**Comment by jonathan1920** [comment]
> @jonathan1920 @helinwang
> Friendly ping. Is this still being worked on? 
Thanks for the reminder! I'll make the changes suggested

**Pull request closed without merging**